content/blog/2023/02/camel-k-roadmap-2023/index.md

```diff
@@ -0,0 +1,92 @@
+
+1. The Camel developer user experience is going to be the same independently on the final deployment choosen
+2. Local development will be easier and straightforward
+3. Something less to maintain... (just a joke, I promise we'll help Camel JBang instead)
+
+Having a generic project, may open some new interesting scenario. For example, at this stage we may be in a better position to support any Camel application, not only Camel Quarkus. And once we're there, the operator could even "intercept" any Camel application deployed by other tools (pipelines, manually, ...) and extend its operations (such as monitoring).
+
+In the same manner, the possibility to leverage the export feature provided by Camel JBang can lead us to a scenario where we can integrate more easily an external pipeline (ie, Tekton) and even think about some opinionated GitOps strategy to adopt.
+
+All these ideas should be furtherly analyzed and refined as they may clash against some assumptions we have already in place (for instance, reusing a container image kit). However, the main concept is that the way we build the application is a strategic piece and we'll be working to make it flexible and more stable in Camel K 2.0.
+
+## Multitenancy
+
+The new multitenancy model development has still some aspect to polish. One of them is the ability to have more than one operator running in the same namespace. We may also think if it makes sense to remove the global operator once we are confident with the new model.
+
+## Observability
+
+We will move from opentracing to opentelemetry, this is something already in progress. As we are focusing on "enterprise" features, probably we should have something more on the Observability aspect, such as providing Service Level KPIs available in the exposed metrics.
+
+## CLI
+
+A lot of work is already done in Camel JBang. We should join effort and move the logic of `kamel` cli in `camel` cli instead. One very important feature we should enhance is the possibility to do a deep code introspection. Right now we can discover simple components and capabilities and it would be a great help to the user experience to have it automatic. In the long run we will be in a position where we hopefully have a full parity feature of what `kamel` offers now into `camel` (therefore having Camel K project in charge only of the operator part).
+
+The benefits of having a common CLI experience from a single tool were previously discussed in the previous chapters.
+
+## Kustomize
+
+So far, we're using the `kamel` CLI also as an intallation method. As we're willing to shrink it, we'll need to work and provide full parity between the actual `kamel install` options and Kustomize.
+
+## Kamelets
```

Comment:
Maybe also mention the initiative to add input/output data types to Kamelets respectively leverage the data types support maybe coming in Camel 4. See #1980